flocker/cli/script.py

```diff
@@ -127,8 +127,14 @@ def fail(msg):
         def got_response(response):
             if response.code != OK:
                 d = json_content(response)
-                d.addCallback(
-                    lambda error: fail(error[u"description"] + u"\n"))
+
+                def got_error(error):
+                    if isinstance(error, dict):
+                        error = error[u"description"] + u"\n"
+                    else:
+                        error = u"Unknown error: " + unicode(error) + "\n"
```

Comment:
This will be an error raised by json_content, right?
Wouldn't it be more useful to have an error handler one level up that printed the full response (including response code and raw response.text) ?